### PR TITLE
Edit User Profile

### PR DESCRIPTION
**Pull request by ahaq0**
Add User Edit, let users edit their profile information e.g. location and update the info in cognito.

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1213/201901.377:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (ahmedhamodi) [inline debug.log:1]: Could probably delete this file.

**Review by ahmedhamodi — APPROVED** [review]
This looks great to me. Thanks again for getting around to doing this.

File: debug.log (hunk 1 above)
Comment (ahaq0) [reply debug.log:1]: is the name of the file directory_reader_win.cc? weird can't grep it

Comment (ahmedhamodi) [reply debug.log:1]: Sorry I meant delete the debug.log file. No idea what the actual log is talking about

**Review by arsidada — APPROVED** [review]

**Pull request merged**